index.html

```diff
@@ -217,39 +226,62 @@ <h2 class="socialmedia">LET'S CONNECT</h2>
                     <li>
                         <div class="language-container">
                             <a class="languages">Languages</a>
-                            <img src="./images/downarrow.svg" alt="" class="arrow" width="40" height="40">
+                            <img src="./images/downarrow.svg" alt="" class="arrow" width="40" height="40" />
                         </div>
                         <ul class="language_items">
                             <li class="javascript">
-                                <img src="./images/js.svg" alt="" class="" width="48" height="48">
+                                <img src="./images/js.svg" alt="" class="" width="48" height="48" />
                                 <a href="#">Javascript</a>
                             </li>
                             <li class="html">
-                                <img src="./images/html.svg" alt="" class="" width="48" height="48">
+                                <img src="./images/html.svg" alt="" class="" width="48" height="48" />
                                 <a href="#">HTML</a>
                             </li>
                             <li class="css">
-                                <img src="./images/css.svg" alt="" class="" width="48" height="48">
+                                <img src="./images/css.svg" alt="" class="" width="48" height="48" />
                                 <a href="#">CSS</a>
                             </li>
                         </ul>
                     </li>
                     <li>
                         <div class="framework-container">
                             <a class="frameworks">Frameworks</a>
-                            <img src="./images/leftarrow.svg" alt="" class="" width="48" height="48">
+                            <img src="./images/leftarrow.svg" alt="" class="" width="48" height="48" />
                         </div>
                     </li>
                     <li>
                         <div class="skills-container">
                             <a class="skills">Skills</a>
-                            <img src="./images/leftarrow.svg" alt="" class="" width="48" height="48">
+                            <img src="./images/leftarrow.svg" alt="" class="" width="48" height="48" />
                         </div>
                     </li>
                 </ul>
             </div>
         </section>
+        <section class="contact-me">
+            <footer class="contact-me-container">
+                <h2 class="contactme-header">Contact me</h2>
+                <p class="">
+                    If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+                </p>
+                <form action="#" id="contact-form" class="contact-form" method="post">
```

Comment:
- [ ] [REQUIRED] Kindly note that the form is required to use the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html). Ensure that the `action` value is similar to `https://formspree.io/f/{form_id}`